nb-api/src/main/java/io/nosqlbench/api/metadata/SystemId.java

```diff
@@ -70,9 +72,21 @@ private static Optional<InetAddress> getMainInetAddrDirect() {
                 if (i2 instanceof Inet4Address) return 1;
                 return 0;
             }).findFirst();
-        return firstInetAddrForInterface;
-    }
 
+        if (firstInetAddrForInterface.orElse(null).getHostAddress().contains(".")) return firstInetAddrForInterface;
```

Comment:
Rephrase this line? This is a potential NPE

Reply:
fixed